.github/workflows/go.yml

```diff
@@ -0,0 +1,47 @@
+on: [push]
```

Comment:
You can add more options to this.  I think push is good.  

I've seen people have pull_request as one option also.  

I'm not sure if you need to change it though.  I've seen people use pretty elaborate rules but we aren't there yet.

Reply:
I have intentionally not pursued things like this yet, including e.g. only running the TS linter if TS files are changed... right now I'm trying to stick to replicating functionality provided by CircleCI.

Reply:
What I was wondering is for github actions, will they run when a PR is merged into main?  

It says push but is a merge considered a push or a pull_request closed?

I'm not familar with circle CI so I don't know if circle CI has sensible defaults.

Reply:
That's a solid question, I'll check that behavior.

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

A merged pull request will trigger the push webhook. So it'll run on push.

(Just saying this out loud) -- We'll need to be more delicate about what we run jobs on once we add releasing to the github actions.